fsharp-backend/tests/Tests/BwdServer.Tests.fs

```diff
@@ -256,198 +248,216 @@ let t filename =
     | None -> ()
 
     // Secrets
-    do!
+    // TODO turn this back into a `do!`
+    let! _ignored =
```

Comment:
Can you add the `do!` back?

Reply:
Done.
For some reason, this was ok within a `testTask {}`:
```
do!
      test.secrets
      |> List.map (fun (name, value) -> LibBackend.Secret.insert meta.id name value)
      |> Task.WhenAll
```

but when extracted to be within a `task {}`, I need to add a `|> Task.map(fun _ -> ())` to the end, to turn the `Task<unit[]>` to a `Task<unit>` whose results can be safely ignored.

If there's a simpler solution here, I'm not aware of it.